neuralpp/symbolic/sympy_expression.py

```diff
@@ -498,3 +499,26 @@ def dict(self) -> Dict[str, Any]:
     @property
     def satisfiability_is_known(self) -> bool:
         return not self._unknown
+
+
+class SymPySummation(SymPyExpression, QuantifierExpression):
```

Comment:
This class is currently not used since SymPy's support of summation is not that general, so the conversion is complicated (e.g., need to split one `BasicSummation` into several `SymPySummation` due to SymPy only supports one interval in `constraint`). But it would be helpful when computing the sum in our future `_eliminate()`.